src/main/java/org/tron/core/net/node/NodeDelegateImpl.java

```diff
@@ -172,7 +172,7 @@ public boolean handleTransaction(TransactionCapsule trx) throws BadTransactionEx
       logger.warn(e.getMessage());
       throw new BadTransactionException();
     } catch (TooBigTransactionResultException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either log or rethrow this exception. ([squid:S1166](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1500&issues=AWXBxdl7ashydt7ke7Fe&open=AWXBxdl7ashydt7ke7Fe)

[ISSUE_KEY]: AWXBxdl7ashydt7ke7Fe